server/views/case-summary-record-order.njk

```diff
@@ -317,75 +317,92 @@
             {% endif %}
 
             <dt class="govuk-!-padding-top-2 govuk-grid-column-one-third pac-!-grid-column-18 govuk-heading-m govuk-!-margin-bottom-0">
-                <h3 class="govuk-body govuk-!-font-weight-bold govuk-!-margin-bottom-0 govuk-!-display-inline-block">Appointments to date</h3>
+                <h3 class="govuk-body govuk-!-font-weight-bold govuk-!-margin-bottom-0 govuk-!-display-inline-block">
+                    Appointments to date</h3>
             </dt>
 
             <dd class="govuk-grid-column-two-thirds govuk-!-margin-0 govuk-heading-m govuk-!-text-align-left govuk-!-padding-top-2 pac-dashboard-count">{{ attendances | length }}</dd>
         </dl>
 
         <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible app-section-break--heavy govuk-!-width-two-thirds">
 
-        <dl class="govuk-grid-row">
-            <div class="govuk-grid-column-one-third pac-!-grid-column-18 govuk-heading-m govuk-!-margin-bottom-0">
-                <h3 class="govuk-body govuk-!-font-weight-bold govuk-!-margin-bottom-0 govuk-!-display-inline-block">Acceptable</h3>
-            </div>
+        <div class="govuk-grid-row">
+            <dl class="govuk-grid-column-one-third pac-!-grid-column-18 govuk-!-margin-top-0">
+                <dt class="govuk-heading-m govuk-!-margin-bottom-0">
+                    <h3 class="govuk-body govuk-!-font-weight-bold govuk-!-margin-bottom-0 govuk-!-display-inline-block"
+                        aria-hidden="true">Acceptable</h3>
+                </dt>
+                <dd hidden="true">Acceptable</dd>
```

Comment:
I think we might have left this in accidentally 🤔 Can these elements be changed back to divs and spans and this line be removed because we've got the one at line 339?

Reply:
yes you right!